cpp/src/arrow/gpu/cuda_context.cc

```diff
@@ -293,6 +293,12 @@ std::shared_ptr<CudaDevice> CudaMemoryManager::cuda_device() const {
   return checked_pointer_cast<CudaDevice>(device_);
 }
 
+Result<std::shared_ptr<DeviceSync>> CudaMemoryManager::MakeDeviceSync(void* sync_event) {
+  return nullptr;
+  // auto ev = reinterpret_cast<CUstream*>(sync_event);
+  // return std::make_shared<CudaDeviceSync>(ev);
```

Comment:
Is this TODO?

Reply:
Yea, in the next PR I'll implement the Cuda implementations ie: a `CudaDeviceSync` object etc.